Actions/CreateApp/action.yaml

```diff
@@ -12,6 +12,10 @@ inputs:
     description: The GitHub token running the action
     required: false
     default: ${{ github.token }}
+  parentTelemetryScope:
```

Comment:
If you want to have the telemetryEventId transferred from the yaml file, then it needs to be a parameter (tests are checking that there is a 1:1 relationship between parameters in .ps1 and .yaml - order, description, required etc)